sdk/communication/azure-communication-identity/tests/test_identifier_raw_id.py

```diff
@@ -319,6 +370,40 @@ def test_equality_based_on_raw_id(self):
             is_anonymous=True,
         )
 
+        # MicrosoftTeamsAppIdentifiers are equal.
+        assert MicrosoftTeamsAppIdentifier(
```

Comment:
pls add assertion for one is setting cloud and the other one is just providing teams app id, and other way around

Reply:
Added.